edb/schema/rewrites.py

```diff
@@ -301,6 +318,13 @@ class RebaseRewrite(
     pass
 
 
+class RenameRewrite(
+    RewriteCommand,
+    referencing.RenameReferencedInheritingObject[Rewrite],
+):
+    pass
```

Comment:
Is this used anywhere or is it here just for consistency?

Reply:
Yeah, if the object or pointer that the rewrite is on gets renamed, the rewrite needs to get renamed too. Everything works without needing to write any real code but the class needs to exist